modules/helfi_react_search/src/Entity/EventList.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\helfi_react_search\Entity;
+
+use Drupal\paragraphs\Entity\Paragraph;
+use Drupal\paragraphs\ParagraphInterface;
+
+/**
+ * Bundle class for Hero -paragraph.
+ */
+class EventList extends Paragraph implements ParagraphInterface {
+
+  /**
+   * Get list of enabled filter keywords.
+   *
+   * @return \Drupal\taxonomy\TermInterface[]
+   *   Enabled keyword.
+   */
+  public function getFilterKeywords() : array {
```

Comment:
This should handle fetching the keywords in the correct language, either by accepting a langcode parameter, getting the correct term translation in the foreach loop or by providing all translations to the React app so it can sort the languages out.